ott/geometry/costs.py

```diff
@@ -118,6 +158,10 @@ def pairwise(self, x: jnp.ndarray, y: jnp.ndarray) -> float:
     cosine_distance = 1.0 - cosine_similarity
     return cosine_distance
 
+  @classmethod
+  def padding_vector(cls, dim: int) -> jnp.ndarray:
+    return jnp.ones((1, dim))
```

Comment:
is this really needed? maybe when ridge is 0? otherwise it feels this somewhat should work.

Reply:
yes, I think we should accommodate for the case where the ridge is 0 and therefore the cost will have NaNs